mina.netty/src/main/java/org/kaazing/mina/core/service/AbstractIoService.java

```diff
@@ -306,7 +306,11 @@ public final void dispose() {
                 try {
                     this.disposalFuture = disposalFuture = dispose0();
```

Comment:
dispose0 returns null in every case as per @stanculescu's investigation. It might have been easier in this case to call the mina ExceptionMonitor directly and not pass through the mina.netty adapter which does that on null session.